examples/imageviewer/mapview-desktop/src/desktopMain/kotlin/example/map/ImageUtils.kt

```diff
@@ -0,0 +1,8 @@
+package example.map
+
+import androidx.compose.ui.graphics.ImageBitmap
+import androidx.compose.ui.graphics.toComposeImageBitmap
+import org.jetbrains.skia.Image
+
+fun ByteArray.toImageBitmap(): ImageBitmap = Image.makeFromEncoded(this).toComposeImageBitmap()
```

Comment:
I already saw a few such functions. Can we reuse previous one?

Reply:
Not for this library